netket/optimizer/qgt/qgt_jacobian_pytree.py

```diff
@@ -66,13 +69,22 @@ def QGTJacobianPyTree(
             **kwargs,
         )
 
+    from netket.vqs.exact import ExactState
```

Comment:
This should be a top-level import.

Reply:
Also, shouldn't this be importable directly from `netket.vqs` like `MCState`?

Reply:
but yes it should bet `nk.vqs.ExactState`

Reply:
But I think this cannot be a module level import because then we have an import loop